module_strings.py

```diff
@@ -2209,77 +2209,77 @@
-  ("spaceship_imperial_star_destroyer_interdictor_desc", "The Interdictor Star Destroyer isn't designed for heavy line combat, ^but it's excellent for tracking medium size ships."),
+  ("spaceship_imperial_star_destroyer_interdictor_desc", "Based on the same design as the Imperial Star Destroyer, the Interdictor^ is faster to address the Imperial-class' poor interception ability."),
   ("spaceship_imperial_dreadnaught_frigate_name", "Imperial Dreadnaught Heavy Cruiser"),
-  ("spaceship_imperial_dreadnaught_frigate_desc", "The Imperial Dreadnought might be outdated, but it fields a great ^fire power and is still respected by both rebels and imperials."),
+  ("spaceship_imperial_dreadnaught_frigate_desc", "The Imperial Dreadnought might be outdated, but it features^ great potential for upgrade in an economical package."),
   ("spaceship_imperial_victory_c2_frigate_name", "Imperial Victory-II Frigate"),
   ("spaceship_imperial_victory_c2_frigate_desc", "The Victory-II Frigate is extensively used by the Empire ^due to its heavy support capabilities."),
   ("spaceship_imperial_trade_frigate_name", "Imperial Trade Frigate"),
-  ("spaceship_imperial_trade_frigate_desc", "The standard Imperial Trade Frigate has a decent base speed ^and lots of space for either troops or cargo."),
-  ("spaceship_tie_fighter_name", "Imperial Tie Fighter"),
-  ("spaceship_tie_fighter_desc", "The Tie Fighter is a relatively modern fighter that makes use of ^the last propulsion technology."),
+  ("spaceship_imperial_trade_frigate_desc", "The standard Imperial Trade Frigate combines acceptable engine speed ^and lots of space for either troops or cargo."),
+  ("spaceship_tie_fighter_name", "Imperial TIE Fighter"),
+  ("spaceship_tie_fighter_desc", "The TIE Fighter is a bare-bones fighter that is only noticeable^ for having the latest in propulsion technology."),
   ("spaceship_imperial_shuttle_name", "Lambda-class T-4a Shuttle"),
   #("spaceship_imperial_shuttle_desc", "A true beauty, the Imperial Shuttle combines speed with troop support capabilites. ^^It has one troop compartment with room for two more and a MK-I Medical bay ^which can be upgraded to MK-II. ^^The current MK-I scanners could be upgraded to MK-III if you require a larger scanning range."),
-  ("spaceship_imperial_shuttle_desc", "A true beauty, the Lambda Shuttle combines speed ^with troop support capabilites."),
+  ("spaceship_imperial_shuttle_desc", "A true beauty, the Lambdap-class Shuttle combines engine speed ^with troop support capabilites and capacity."),

```

Comment:
@airtroop392 I think that _lambdap_  is a typo, I can correct it later. Everything until here looks pretty good.